src/file-operations.js

```diff
@@ -1,6 +1,7 @@
 const path = require("path");
 const fs = require("mz/fs");
 const compareVersions = require("compare-versions");
+const validate = require("compare-versions");
```

Comment:
@dlowder-salesforce There looks to be a duplicate import here. This package is being imported above.